support/windows-client/networking/tcp-ip-packet-loss-troubleshooting.md

```diff
@@ -0,0 +1,105 @@
+# Diagnosing Packet Loss on Windows
+
+## Overview
+
+Packet loss occurs whenever a network packet does not reach its intended destination. Some packet loss is normal, and a dropped packet does not automatically imply it is the cause of a higher-level networking issue. At other times, packet loss can cause reduced performance, and APIs or even applications to fail.
+
+## Capturing Packet Loss Diagnostics
+
+The first step when conducting a packet loss investigation is to capture [pktmon](https://learn.microsoft.com/en-us/windows-server/networking/technologies/pktmon/pktmon) traces, typically by using the pktmon command-line workflow. Pktmon is capable of capturing packet traces, attributing local packet loss to specific reasons and code locations, and collecting packet loss statistics. When combined with [Wireshark](https://www.wireshark.org/) analysis of protocol-level behavior, pktmon traces are sufficient to root cause many cases of packet loss. Pktmon will also be capable of collecting packet loss statistics within network cards, but as of early 2025, this feature is not available.
+
+If pktmon diagnostics are inconclusive, more exhaustive component-level traces may be captured using netsh.exe trace start scenario=InternetClient or netsh.exe trace start scenario=InternetServer for client and server scenarios respectively, followed by netsh.exe trace stop after the events have been captured. These component-level traces are relatively noisy and opaque, but often contain additional context before or after a local packet is dropped, or for remote loss, an indication the local system has inferred packet loss has occurred. The traces can be converted to text using netsh.exe trace convert nettrace.etl, opened in [Windows Performance Analyzer](https://learn.microsoft.com/en-us/windows-hardware/test/wpt/windows-performance-analyzer), or used with any other ETW tool.
```

Comment:
netsh.exe commands should be in marked with quotes to look like code, or maybe, even broken to their own Powershell code block.

It is pretty hard to parse the sentence right now with the command not being delimited in any form.

Reply:
Oh, these didn't get auto-converted to .md properly from my .docx. I'll fix that.